constants.js

```diff
@@ -9,4 +9,6 @@ const DEFAULT_DEPTH = 1;
 const DEFAULT_STORY_FIELDS =
   "headline,subheadline,slug,url,hero-image-s3-key,hero-image-caption,hero-image-metadata,first-published-at,last-published-at,alternative,published-at,authors,author-name,author-id,sections,story-template,metadata,access,access-level-value,read-time";
 
-module.exports = { DEFAULT_DEPTH, DEFAULT_LIMIT, DEFAULT_STORY_FIELDS };
+const DEFAULT_REQUEST_TIMEOUT = 2000;
```

Comment:
On what basis is this 2 seconds?

Reply:
The response rate is to be < 700s, worse case scenario is to be < 2s